### PR TITLE
fix: example thumbnail build

### PR DESCRIPTION
**Pull request by epreston**
The file and content exists in the generated site content but the class for this example is commented out.  This bocks thumbnails from being generated for the whole project.

Related: #5521

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

**Comment by willeastcott** [comment]
Hey @mvaligursky and @slimbuck, I know this example was hidden because it's not ready for public comsumption. But if it's breaking the build, we should either merge this or figure out a way to hide it without breaking stuff. I'll leave it to you to two to decide.

**Comment by epreston** [comment]
It's not being hidden when you build examples.  It is there in the list of examples, just no thumbnail. The thumbnail builds break halfway through leaving only half thumbs.

**Comment by mvaligursky** [comment]
For now we need to remove the example (instead of commenting it out). It works, but the API is not finalised / public yet.
We used to have HIDDEN option in the example being supported before, to skip adding those examples to the list before, but that seems gone now, so we just need to remove its import for now.

**Comment by mvaligursky** [comment]
Yep, we should just remove those two lines with LitMaterialExample in them from the index.mjs

**Comment by epreston** [comment]
The thumbnails are built by enumerating files in a directory.

**Comment by mvaligursky** [comment]
Ah I see. Hmm, this feels pretty hidden way to do this, I can see somebody spending time trying to figure out why this example does not have a thumbnail in the future. Perhaps add a log when this name is filtered out, so that we're aware, thanks.

File: examples/src/examples/graphics/index.mjs [hunk 1]
```diff
@@ -69,7 +69,7 @@ export {
     ClusteredOmniShadowsExample,
     ClusteredSpotShadowsExample,
     ContactHardeningShadowsExample,
-    // LitMaterialExample,
+    LitMaterialExample,
```
Comment (mvaligursky) [inline examples/src/examples/graphics/index.mjs:72]: and here please remove this line, and also the relevant import in the same file - the example is not ready to be shipped yet.

Comment (epreston) [reply examples/src/examples/graphics/index.mjs:72]: No worries.

**Comment by epreston** [comment]
I'm knee deep in the examples build stuff.  I'd like to add that HIDDEN option back in, would a PR for that be welcome ?

File: examples/src/examples/graphics/index.mjs (hunk 1 above)
Comment (epreston) [reply examples/src/examples/graphics/index.mjs:72]: Remove or comment ?  So you can quickly undo this ?

**Comment by mvaligursky** [comment]
> I'd like to add that HIDDEN option back in, would a PR for that be welcome

It would be very much appreciated to have it back, thanks for looking at this!

**Review by mvaligursky — APPROVED** [review]
Perfect, thanks, commenting out is great.

**Pull request merged**